pkg/syncthing/process.go

```diff
@@ -14,12 +14,37 @@
 package syncthing
 
 import (
+	"runtime"
 	"time"
 
 	"github.com/shirou/gopsutil/process"
 )
 
 func terminate(p *process.Process, wait bool) error {
+	if runtime.GOOS == "windows" {
+		children, err := p.Children()
+		if err != nil {
+			return err
+		}
+		err = terminateProccess(p, wait)
+		if err != nil {
+			return err
+		}
+		for _, child := range children {
+			err = terminateProccess(child, wait)
+			if err != nil {
+				return err
+			}
+		}
+		return err
+	}
+	err := terminateProccess(p, wait)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+func terminateProccess(p *process.Process, wait bool) error {
```

Comment:
I think you still need to phases in hard terminate, or add this logic there too. If not, is there are zombie syncthing instances for what ever reason, hardTerminate wouldn't kill all of them

Reply:
This code it's also affecting to hard terminate. It's the code that you call [here](https://github.com/okteto/okteto/blob/96eed3f63d5be1b8fab99cc2f21efe2c5fd753fd/pkg/syncthing/syncthing.go#L688)

Reply:
Ah, perfect!